modules/javafx.graphics/src/test/java/test/javafx/scene/TreeShowingPropertyTest.java

```diff
@@ -33,66 +33,42 @@
 import javafx.scene.SubScene;
 import javafx.scene.layout.StackPane;
 import javafx.stage.Stage;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-import org.junit.runners.Parameterized.Parameters;
 
-import java.util.Arrays;
-import java.util.Collection;
+import java.util.stream.Stream;
 import java.util.concurrent.atomic.AtomicReference;
-import java.util.function.Supplier;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
-@RunWith(Parameterized.class)
 public class TreeShowingPropertyTest {
-    private final Parent root;
-    private final Node node;
-    private final TreeShowingProperty property;
-
-    @Parameters
-    public static Collection<Object[]> parameters() {
-        Supplier<RootAndNodeToTest> supplier1 = () -> {
-            Node node = new StackPane();
-            return new RootAndNodeToTest(new StackPane(node), node);
-        };
-
-        Supplier<RootAndNodeToTest> supplier2 = () -> {
-            StackPane node = new StackPane();
-            return new RootAndNodeToTest(new StackPane(new SubScene(node, 100.0, 100.0)), node);
-        };
-
-        return Arrays.asList(new Object[][] { { supplier1 }, { supplier2 } });
```

Comment:
the change may not be fully equivalent: the use of `Supplier` defers the moment of creation until each test is started.

it is very likely that the parameters() method is called once for each test, creating all the instances before any of the tests cases are executed.

this still might be ok because 
a) the tests are headless
b) the nodes are not inserted into the scene graph
c) instances are not reused anywhere

Reply:
You're right, someone probably did this originally for a reason. I didn't know `Supplier` would work like this, but the more you know!

Reverted to how it used to be, but JUnit5-ified.